src/controllers/notice.expired.controller.ts

```diff
@@ -5,7 +5,7 @@ import {
   retrieveCompanyProfileFromObjectionSession,
 } from "../services/objection.session.service";
 
-export const get = async (req: Request, res: Response, next: NextFunction) => {
+export const get = (req: Request, res: Response, next: NextFunction) => {
```

Comment:
Lint was showing up the presence of 'async' as not needed/a violation. Removed it and all seems to still work.

Reply:
you can probably remove NextFunction as well and its import